### ISSUE TITLE
Which MySQL server to choose ?

### ISSUE THREAD

**Issue by Meroje** [Meroje/puppet-devbox#3]
There's plenty of reason to and to not use the "official" oracle mysql server.
And there's very good quality forks out there, which one to choose ?

So far Im aware of percona and mariadb. Thoughts ? Advice ?

**Comment by niallobrien** [comment]
Personally, I'd be inclined to recommend Postgres.
"Currently Laravel supports four database systems: MySQL, Postgres, SQLite, and SQL Server."

**Comment by Meroje** [comment]
I don't think I want to do the choice of not including mysql or postgres, I will include both but with a configuration switch. What do you think ?

**Comment by niallobrien** [comment]
Fair enough. Most php developers would be used to working with mySQL, so I'd say include that as the default.
